index.html

```diff
@@ -12,7 +12,15 @@
 <body>
   <div class="body-container">
     <h2 class="main-title">Front-End Tools</h2>
-    <h2 class="main-title">Back-End Tools</h2>
+    <div class="tech">
```

Comment:
If this is going to contain all the techs I'd name this `techs-container` and I'd change each tech fron `tech-container` to `tech`